modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -1054,6 +1086,34 @@ definition {
 		return ${entryId};
 	}
 
+	@summary = "Get the external reference code of an object field"
+	macro getObjectFieldERC(objectName, fieldName) {
```

Comment:
This macro is necessary because, in order to add partial validations, it is necessary to [indicate the field to be used with the external reference code](https://github.com/liferay-bpm-qa/liferay-portal/pull/1211/files#diff-6895477eded1044eb00bb86ea6541c60b7b6e65b5dc91dab01a073a20c9e97bdR658)